astilectron.go

```diff
@@ -170,15 +176,14 @@ func (a *Astilectron) Start() (err error) {
 		}
 	}
 
-	// Unfortunately communicating with Electron through stdin/stdout doesn't work on Windows so all communications
-	// will be done through TCP
-	if err = a.listenTCP(); err != nil {
-		return errors.Wrap(err, "listening failed")
+	listenType, listenErr := a.listen()
```

Comment:
Revert this part back to the original version and just rename the function from `listenTCP` to `listen`